src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,428 @@
+| Phase-contrast microscopy                     | PC       |
+| Differential interference contrast microscopy | DIC      |
+| Fluorescence microscopy                       | FLUO     |
+| Confocal microscopy                           | CONF     |
+| Polarized-light microscopy                    | PLI      |
+| Coherent anti-Stokes Raman spectroscopy       | CARS     |
+| 2-photon excitation microscopy                | 2PE      |
+| Multi-photon excitation microscopy            | MPE      |
+| Super-resolution microscopy                   | SR       |
+| Nonlinear optical microscopy                  | NLO      |
+| Optical coherence tomography                  | OCT      |
+| Selective plane illumination microscopy       | SPIM     |
+
+### Filename entities
+
+In the context of Microscopy, a session ([`ses-<label>`](../99-appendices/09-entities.md#ses))
+can refer to all the acquisitions between the start and the end of an imaging experiment
+for ex vivo imaging or a subject lab visit for biopsy procedure and/or in vivo imaging.
+
+The [`sample-<label>`](../99-appendices/09-entities.md#sample) entity is REQUIRED for
+Microscopy data and is used to distinguish between different samples from the same subject.
+The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example:
+Three brain slices (`sample-01` to `sample-03`) extracted from subject `sub-01`.
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_<modality_suffix>.<ext>
+        sub-01_sample-02_<modality_suffix>.<ext>
+        sub-01_sample-03_<modality_suffix>.<ext>
+        sub-01_<modality_suffix>.json
+```
+In this example, the JSON metadata is common for all samples of `sub-01`.
+JSON metadata may be defined per subject or per sample as appropriate, as per the
+[inheritance principle](../02-common-principles.md#the-inheritance-principle).
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity corresponds to a custom label that
+MAY be used to distinguish a different set of parameters used for acquiring the same modality.
+For example, two images of the same sample acquired with different magnification of 40x and 60x.
+In such case two files could have the following names:
+`sub-01_sample-01_acq-40x_<modality_suffix>.<ext>` and
+`sub-01_sample-01_acq-60x_<modality_suffix>.<ext>`, however the user is free to choose any other
+label as long as they are consistent across subjects and sessions.
+
+The [`stain-<label>`](../99-appendices/09-entities.md#stain) entity can be used to distinguish
+image files from the same sample using different stains or antibodies for contrast enhancement.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example:
+One brain slice (`sample-01`) extracted from subject `sub-01`, imaged with three stains
```

Comment:
```suggestion
For example: one brain slice (`sample-01`) extracted from subject `sub-01`, imaged with three stains
```
unless want into a separate paragraph -- then an empty line is needed. But I have chosen above suggestion as matching the examples I found in current BIDS

Reply:
Thank you for pointing this out, the intention was to have everything in one paragraph as you suggested. I corrected this instance and 2 others in bfb027b.